src/Linux/Avalonia.LinuxFramebuffer/Output/DrmOutput.cs

```diff
@@ -161,11 +174,22 @@ uint GetCrtc()
             if(_gbmTargetSurface == IntPtr.Zero)
                 throw new InvalidOperationException("Unable to create GBM surface");
 
-            _eglDisplay = new EglDisplay(new EglInterface(eglGetProcAddress), false, 0x31D7, device, null);
-            _platformGl = new EglPlatformOpenGlInterface(_eglDisplay);
-            _eglSurface =  _platformGl.CreateWindowSurface(_gbmTargetSurface);
+            _eglDisplay = new EglDisplay(
+                new EglDisplayCreationOptions
+                {
+                    Egl = new EglInterface(eglGetProcAddress),
+                    PlatformType = 0x31D7,
+                    PlatformDisplay = device,
+                    SupportsMultipleContexts = true,
```

Comment:
API-wise, what's a point of `SupportsMultipleContexts = true` when `IPlatformGraphics.CreateContext` always throws an exception?